<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /fs/turing/cvs/copasi_dev/copasi/UI/CQNotes.ui,v $ </comment>
<comment>   $Revision: 1.5 $ </comment>
<comment>   $Name: Build-33 $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2010/08/18 17:33:04 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2010 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQNotes</class>
 <widget class="CopasiWidget" name="CQNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWebView" name="mpWebView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="mpEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="mpBtnToggleEdit">
     <property name="text">
      <string>...</string>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="mpSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>151</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">copasi/UI/copasiWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnToggleEdit</sender>
   <signal>clicked()</signal>
   <receiver>CQNotes</receiver>
   <slot>slotToggleMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpWebView</sender>
   <signal>linkClicked(QUrl)</signal>
   <receiver>CQNotes</receiver>
   <slot>slotOpenUrl(QUrl)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEdit</sender>
   <signal>textChanged()</signal>
   <receiver>CQNotes</receiver>
   <slot>slotValidateXML()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
